reference/forms/types/entity.rst

```diff
@@ -112,12 +112,10 @@ The class of your entity (e.g. ``AcmeStoreBundle:Category``). This can be
 a fully-qualified class name (e.g. ``Acme\StoreBundle\Entity\Category``)
 or the short alias name (as shown prior).
 
-.. include:: /reference/forms/types/options/data_class.rst.inc
-
 em
 ~~
 
-**type**: ``string`` **default**: the default entity manager
+**type**: ``string`` **default**: the default entity manager class
```

Comment:
Well, this option can be a string. But then it is the manager name not the class name. Additionally, you can also pass . `ObjectManager` instances here.